### DISCUSSION TITLE
Snapshot issue with WYSIWYG field with mssql database

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#13951] [Archive]
Hi @rijkvanzanten and team,

I am trying to apply snapshot with field type WYSIWYG considering max as -0.5 and getting error.
We are using database mssql, and directus version 9.8.0

**Comment by azrikahar** [comment]
This should be the same issue as #13837 and it was recently fixed by #13845 which will be released in the next version (current version is 9.12.2).

**Reply by sandyksit** [reply]
Thanks @azrikahar
